assignments/objects.js

```diff
@@ -49,16 +86,35 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name": "Susan",
+  "age": "70"
+}
 
-// Log the parent object's name
+parent.child = {
+  "name": "George",
+  "age": "50"
+}
 
-// Log the child's age
+parent.child.grandchild = {
+  "name": "Sam",
+  "age": "30"
+}
 
-// Log the name and age of the grandchild
+parent.speak = "Hello, my name is Susan";
+parent.child.speak = "Hello, my name is George";
+parent.child.grandchild.speak = "Hello, my name is Sam";
+console.log(parent);
 
+// Log the parent object's name
+console.log(parent.name);
+// Log the child's age
+console.log(parent.child.age);
+// Log the name and age of the grandchild
+console.log(parent.child.grandchild.name + ", " + parent.child.grandchild.age)
 // Have the parent speak
-
+console.log(parent.speak);
```

Comment:
```suggestion
console.log(parent.speak());
```
And do the same for child and grandchild